### PR TITLE
Fix missing documentation for pluggable widgets page

### PR DESCRIPTION
**Pull request by mijay**

### CODE REVIEW

**Comment by r0b1n** [comment]
WIP

**Comment by ConnorLand** [comment]
> WIP. Do not merge yet

Understood.

**Comment by mijay** [comment]
@ConnorLand we are ready now

**Comment by ConnorLand** [comment]
> @ConnorLand we are ready now

Ok. I'll proof things and then merge. Thanks all.

File: content/apidocs-mxsdk/apidocs/property-types-pluggable-widgets.md [hunk 1]
```diff
@@ -299,14 +300,18 @@ The widgets property allows a user to place multiple widgets inside a pluggable
 This property type was introduced in Mendix 8.3.
 {{% /alert %}}
 
-#### 3.1.1 XML Attributes
+{{% alert type="warning" %}}
+Some widgets are not yet supported inside pluggable widgets. Placing unsupported widgets inside a pluggable widget results in a consistency error in Studio and Studio Pro.
```
Comment (ConnorLand) [inline content/apidocs-mxsdk/apidocs/property-types-pluggable-widgets.md]: @mijay do we have widget-by-widget info on which are supported to go inside pluggable widgets? Can we tell users which are/are not supported? If so, where?

Comment (mijay) [reply content/apidocs-mxsdk/apidocs/property-types-pluggable-widgets.md]: We have, but it does change quite frequently, so it is fairly annoying to keep this list here and change it with every release...

Comment (ConnorLand) [reply content/apidocs-mxsdk/apidocs/property-types-pluggable-widgets.md]: Ok, I see.

**Pull request merged**